stdlib/mlang/main.mc

```diff
@@ -0,0 +1,108 @@
+include "result.mc"
+include "fileutils.mc"
+include "compile.mc"
+-- include "../../src/main/mi-lite.mc"
+-- include "../../src/main/compile.mc"
+-- include "../../src/main/options.mc"
+include "sys.mc"
+
+include "compile.mc"
+include "boot-parser.mc"
+include "ast-builder.mc"
+include "symbolize.mc"
+include "composition-check.mc"
+include "include-handler.mc"
+include "language-composer.mc"
+include "const-transformer.mc"
+
+include "mexpr/eval.mc"
+include "mexpr/builtin.mc"
+include "mexpr/ast-builder.mc"
+include "mexpr/phase-stats.mc"
+include "mexpr/pprint.mc"
+
+lang MainLang = MLangCompiler + BootParserMLang + 
+                MLangSym + MLangCompositionCheck +
+                MExprPrettyPrint + MExprEval + MExprEq + 
+                MLangConstTransformer + MLangIncludeHandler +
+                PhaseStats + LanguageComposer
+
+  sem myEval : Expr -> Expr
+  sem myEval =| e ->
+    eval (evalCtxEmpty ()) e 
+
+  -- TODO: add node count for MLang programs to phase-stats
```

Comment:
I ran into the same issue in some other places as well, which suggest that phase-stats maybe shouldn't assume things to be `Expr`, rather take a function or something like that.

Also, why is this code commented out but present?

Reply:
Yeah, it would be good if phase-stats was a bit more general.

Reply:
The code for MLang became outdated. I'll re-add it at some point, but for now I've removed the comment.